myuw/test/api/instructor_schedule.py

```diff
@@ -12,31 +12,49 @@
 FDAO_Canvas = 'restclients.dao_implementation.canvas.File'
 
 
-@require_url('myuw_instructor_schedule')
-class TestInstructorSchedule(MyuwApiTest):
+@require_url('myuw_instructor_current_schedule_api')
+class TestInstructorCurrentSchedule(MyuwApiTest):
     def test_bill_current_term(self):
-        with self.settings(RESTCLIENTS_SWS_DAO_CLASS=FDAO_SWS,
-                           RESTCLIENTS_PWS_DAO_CLASS=FDAO_PWS,
```

Comment:
I see that MyuwApiTest does the SWS override, but I don't see the pws or canvas overrides in there.